src/main/java/uk/gov/hmcts/reform/sscs/helpers/CaseFlagMappingServiceHelper.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.hmcts.reform.sscs.helpers;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+@SuppressWarnings("PMD")
+public class CaseFlagMappingServiceHelper {
+
+    private final String[] mappingValues =
+        {
+        "disabledAccess",
+        "signLanguageType",
+        "hearingLoop",
+        "isConfidentialCase",
+        "dwpUCB",
+        "dwpPHME",
+        "urgentCase",
+        "adjournCaseInterpreterLanguage"
+        };
+
+    private final Map<String, Object> caseFlags;
+
+    public CaseFlagMappingServiceHelper() {
+        this.caseFlags = new HashMap<>();
```

Comment:
Interesting use of state.. I think it's okay.

Reply:
I wouldn't hold state on the helper either. You should pass in the data structure when you call mapCaseFlags